### DISCUSSION TITLE
Thumbs with centeredSlides

### DISCUSSION THREAD

**Discussion by Arcadise** [nolimits4web/swiper#6984] [Q&A]
Hello,
I'm currently using Thumbs for a personal project, and I started from the example in the documentation, which works great. However, I wanted the slide to always be centered on the Thumbs, but I'm not sure how to achieve that. If I could get some assistance, here's where I'm at: https://codesandbox.io/p/sandbox/swiper-thumbs-gallery-forked-wfhk2n?file=/index.html:179,33

**Comment by Arhell** [comment]
Hi try https://www.sitepoint.com/css-viewport-units-quick-start/#fullscreenbackgroundimagesorsections
maybe it will work, unfortunately the project did not load

**Reply by Arcadise** [reply]
oh, to reproduce you juste need to take the 'Thumbs gallery' demo and get something like that for the swiper config 
` var swiper = new Swiper(".mySwiper", {
      spaceBetween: 10,
      slidesPerView: 5,
      freeMode: true,
      watchSlidesProgress: true,
      centeredSlides: true,
      loop : true,
    });
    var swiper2 = new Swiper(".mySwiper2", {
      spaceBetween: 10,
      loop: true,
      navigation: {
        nextEl: ".swiper-button-next",
        prevEl: ".swiper-button-prev",
      },
      thumbs: {
        swiper: swiper,
      },
    });`

**Reply by Arhell** [reply]
Hmm, if you center it, maybe it will help
https://swiperjs.com/demos#centered
https://swiperjs.com/demos#centered-auto
If I understand you correctly

**Reply by Arcadise** [reply]
This is what I did, but the centered doesn't work well with Tumbs, the active Thumbs is not always at the center

**Reply by Arhell** [reply]
https://github.com/nolimits4web/swiper/issues/3851 but this is an old issue

**Reply by Arcadise** [reply]
I saw it, but there is no solution

**Reply by Arhell** [reply]
Clearly, then  slider works like the example in the documentation?
You can expand the image to full width or center the slider itself in the center if there are gaps at the edges

**Reply by Arhell** [reply]
https://codepen.io/intotheprogram/pen/PowvWeX
